osu.Game/Overlays/ChatOverlayV2.cs

```diff
@@ -215,7 +219,7 @@ protected override void OnDrag(DragEvent e)
                 return;
 
             float targetChatHeight = dragStartChatHeight - (e.MousePosition.Y - e.MouseDownPosition.Y) / Parent.DrawSize.Y;
-            chatHeight.Value = targetChatHeight;
+            chatHeight.Value = Math.Clamp(targetChatHeight, min_chat_height, max_chat_height);
```

Comment:
This is already enforced on the config bindable:

https://github.com/ppy/osu/blob/e2895ea1459118e1e20323d548f06c45df4f8c72/osu.Game/Configuration/OsuConfigManager.cs#L47

The bigger problem here is that `chatHeight` is a `Bindable<float>` rather than a `BindableFloat`. The difference between the two is that the latter is actually a `BindableNumber<float>`, which has min and max constraints built in. If you bind that to the config bindable, this clamp should no longer be required.

Related: https://github.com/ppy/osu-framework/issues/3229

Reply:
Makes sense.
Incidentally, using the correct bindable type also seems to fix the test failures.